cli/submit.go

```diff
@@ -93,9 +105,10 @@ func SubmitRun(_ *cmd.RootCMD, c *cmd.CMD) {
 // a pull request instead of pushing directly to their repo.
 func promptDoPullRequest(url string) bool {
 	fmt.Printf(
-		`You don't appear to have write permissions for the %v.
-Do you want to submit a pull request to the repository instead? This is the 
-only way to continue the submission. (y/[n]) `, url)
+		`You don't appear to have write permissions for the 
```

Comment:
Could you get rid of the `the` at the end of this line please? Looks like I was mashing sentences when I wrote this printout.